packages/api-client/src/conversation/Conversation.ts

```diff
@@ -91,4 +91,6 @@ export interface Conversation {
 
   /** MLS conversations only */
   group_id?: string;
+
+  protocol: ConversationProtocol;
```

Comment:
should this be an optional param? wondering about backwards compatability

Reply:
Good catch. But the backend is sending this value already on the production server. So it is not optional anymore.

Reply:
ok then